src/interpreter.ts

```diff
@@ -1133,7 +1133,8 @@ export class Interpreter {
                         this.interpretExpression(ast.args[0]!),
                         ast.args[0]!.loc,
                     );
-                    return BigInt(crc32.str(str));
+                    const c = BigInt(crc32.str(str));
+                    return c < 0 ? c + 4294967296n : c;
```

Comment:
https://github.com/SheetJS/js-crc32?tab=readme-ov-file#signed-integers:

> Unconventional for a CRC32 checksum, this library uses signed 32-bit integers. This is for performance reasons. 

> `x >>> 0` converts a number value to unsigned 32-bit integer.

Alternatively, we could use AWS's implementation of CRC-32 which seems to use unsigned numbers: https://www.npmjs.com/package/@aws-crypto/crc32.

Reply:
If you decide to keep js-crc32, then let's switch to `x >>> 0` and add this comment about signed numbers, otherwise it's a bit unclear

Reply:
done